### ISSUE TITLE
Changing nrf24l01 channel

### ISSUE THREAD

**Issue by jjmz** [kehribar/nrf24L01_plus#2]
One user (Bammetje, probably the only one) of the code I compiled on STC15L204 boards (http://jjmz.free.fr/?p=179) and which is using your code for the nrf24l01 has submitted this comment to me :

> > Got it!!
> > The channel should be set after the other parameters. I put the set channel 
> > instruction just before the poweruprx line in nrf24.c It is now on line 63 and working 
> > fine. Maybe you can put it there in your codebase as well, just in case people want 
> > to change the channels.

This is probably also true for your code.

JJ

**Comment by electrodonkey** [comment]
i agree with the above comment. nrf24_tx_address() cannot write the tx_address register if the chip is not powered down
